integrations/paddleocr/pydoc/config.yml

```diff
@@ -0,0 +1,29 @@
+loaders:
```

Comment:
This is no longer needed. Let's remove this file and keep `config_docusaurus.yml` only.